data/scripts/sql_loader.py

```diff
@@ -1,44 +1,182 @@
-# Work in progress
-# TODO: Consolidate with O*Net and other data sources
-# TODO: SQLite + Postgres support (Cloud?)
-# TODO: Better organization + classes
+# TODO: Abstract data cleaning step(s) and connection layer
+# TODO: Change sqlalchemyengine creds based on prod location
+
+# Requires jobhopper base directory to be added to PATH in venv/bin/activate
+# Start postgres server first (default for macOS) pg_ctl -D /usr/local/var/postgres start
 
-# Requires jobhopper to be added to PATH in venv/bin/activate
-from data.bls.oes_data_downloader import OESDataDownloader
-import sqlite3
 import logging
+import os
+
+import pandas as pd
+
+from sqlalchemy.types import Integer, Numeric, String
+from sqlalchemy import create_engine
+from data.bls.oes_data_downloader import OESDataDownloader
+from data.bls.utils.dtype_conversion import to_float, to_int
+
 
+logging.basicConfig()
 log = logging.getLogger()
 log.setLevel(logging.INFO)
 
```

Comment:
Note for readme: When environment variables are not set, this will only work if people have set up a postgres user with username `jobuser` and password `jobuser` with the other specifications from the readme setup